.husky/pre-commit

```diff
@@ -4,12 +4,6 @@
 cd apps/nowcasting-app || exit
 yarn lint
 
-# check if the server is already running and save the boolean result
-serverRunning=$(lsof -t -i :3002)
-# if the server is running, just run the tests and don't start the server
-if [ -z "$serverRunning" ]; then
-  yarn test
-else
-  # if the server isn't running, start it, run the tests, and then kill the server after the tests are done
-  yarn dev & yarn test && lsof -t -i :3002 | xargs kill
-fi
+# Now only runs unit tests, Cypress tests are run in the CI pipeline on PRs
+yarn test
+
```

Comment:
do unit tests get run in CI too?

Reply:
Yep 👍 both sets of tests run in CI, but the faster unit test suite runs pre-commit